releaseNotes/AzureDevOpsPushPipelineExtension.md

```diff
@@ -2,6 +2,13 @@
 Azure App Configuration Push extension for Azure DevOps pipeline can be installed from the [Azure DevOps marketplace](https://marketplace.visualstudio.com/items?itemName=AzureAppConfiguration.azure-app-configuration-task-push). The version information can be found in the console when the task is executed:
 ![sample](pictures/AzureDevOpsPushExtensionVersionSample.PNG)
 
+### v4.4.0 - November, 15 2022
+* Added Import mode option support for KVSet profile.

```

Comment:
I see "Import mode", "ImportMode", and "Import Mode" in this PR. Can we make up our minds about which format to go with? :)

Reply:
Apologies for that. Updated to ImportMode to be consistent with the way the property name should be written in YAML files